lang/he/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Semantic Versioning 2.0.0
+language: he
+language_dir: rtl
+---
+
+גירסאות סמנטיות 2.0.0
+==============================
+
+סיכום
+-------
+
+בהינתן מספר גרסה MAJOR.MINOR.PATCH, העלה את:
+
+1. גרסה ראשית (MAJOR) כאשר אתה עושה שינויים שאינם עולים בקנה אחד עם ה API,
+1. גרסה משנית (MINOR) כשאר אתה מוסיף פונקציונאליות בצורה שמותאמת אחורה, ו
+1. גרסת תיקון (PATCH) כאשר אתה מבצע תיקוני באגים שמותאמים אחורה.
```

Comment:
Yes, I think it's github markdown flavor, but it's handle it. It'll output 
```
1. ...
2. ...
3. ...
```

All the other version write it this way, so I stay consistent